### PR TITLE
Adding lines for basic nginx config

### PR DESCRIPTION
**Pull request by jkochis**
I made a couple edits to be less Apache-centric, and added the lines for nginx config.

### CODE REVIEW

File: docs/tutorial/first-app.md [hunk 1]
```diff
@@ -104,6 +113,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule . index.php [L]
 ```
 
+nginx does not use `.htaccess` files, so you will need to add the following to your server configuration in the `location` block:
+
+```
+if (!-e $request_filename){
+    rewrite ^(.*)$ /index.php break;
```
Comment (geggleto) [inline docs/tutorial/first-app.md:120]: Can you add a comment about how `index.php` is actually whatever entry point into your application... I know some people don't use index.php instead use something like api.php

Comment (jkochis) [reply docs/tutorial/first-app.md:120]: Sure, but before that in the tutorial it tells you to create an index.php:

https://github.com/jkochis/Slim-Website/blob/gh-pages/docs/tutorial/first-app.md#create-the-application

I'm glad to contribute in any way I can, but I don't want to alter the docs too much from the original. My goal was to simply add lines for nginx users. I feel adding that could be TMI. What do you think?

Comment (geggleto) [reply docs/tutorial/first-app.md:120]: Well I suppose if people are changing the entry point, we could assume they know to change the nginx config :)

Comment (jkochis) [reply docs/tutorial/first-app.md:120]: Sounds good, my goal was just a few nginx 'subtitles' for the existing Apache config in the tutorial.

Comment (jkochis) [reply docs/tutorial/first-app.md:120]: Added!

**Comment by akrabat** [comment]
Thank you.

**Pull request merged**